web/impact/impact/tests/test_create_edit_office_hours_view.py

```diff
@@ -216,6 +217,64 @@ def test_admin_cant_create_office_hour_for_non_mentor_user(self):
         with self._assert_office_hour_created(created=False):
             self._create_office_hour_session(self.staff_user(), data)
 
+    def test_mentor_cant_create_conflicting_sessions(self):
```

Comment:
This test fails if we start the new sessions at `start_time + timedelta(hours=2)`

Reply:
That is the expected behavior. The test is expecting no office hours to be created since there will be conflicts when `start_date_time` = `start_time`. However, when `start_date_time` = `start_time + timedelta(hours=2)` the office hour is created because there no conflicts which fails the test

Reply:
Okay. In that case, let's write a test that explicitly tests that. 

For example, add a `block_duration=30` parameter to `_create_office_hour` and set `end_time=start_time+timedelta(block_duration)`

Then you can `self._create_office_hour(start_date_time=now, duration=120)` and request a block starting at `now + timedelta(60)`